src/proto/flwr/proto/exec.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Flower Labs GmbH. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ==============================================================================
+
+syntax = "proto3";
+
+package flwr.proto;
+
+service Exec {
+  // Start run upon request
+  rpc StartRun(StartRunRequest) returns (StartRunResponse) {}
+}
+
+message StartRunRequest { bytes fab_file = 1; }
```

Comment:
Should we add the file name here?

Reply:
@danieljanes I don't think we will need it anywhere, what would be the reason to have it?

Reply:
For example, if we want to show to the user at some point the files they submitted, it'd be good to show them with the same filename they have seen locally. But ok to not have it in this first version - resolved.